articles/_posts/2015-07-23-making-animated-themes.md

```diff
@@ -22,17 +22,21 @@ In the video below, you can see an example of animated themes in action.
 </figure>
 
 Videos by [NatureClip](http://natureclip.co.nr/) Licensed under CC-BY
-[Silent Forest]({{ page.id }}/forest.zip) (10 MB) & [Still Lake]({{ page.id }}/still-lake.zip) (6 MB)
+[Still Lake]({{ page.id }}/still-lake.zip) (6 MB) & [Silent Forest]({{ page.id }}/forest.zip) (10 MB).
```

Comment:
Could you please expand this part a bit? Something simple, like: _download and install extensions like shown on video, thanks to NatureClip for videos, etc._